app/src/main/scala/org/alephium/explorer/error/ExplorerError.scala

```diff
@@ -37,22 +37,27 @@ sealed trait FatalSystemExit extends ExplorerError
 object ExplorerError {
 
   /******** Group: [[FatalSystemExit]] ********/
-  final case class FailedToFetchSelfClique(remoteMessage: String)
-      extends Exception(s"Could not fetch self-clique: $remoteMessage.")
+  final case class UnreachableNode(cause: Throwable)
+      extends Exception(s"Could not reach node: $cause.")
```

Comment:
I'm not sure how exceptions in Strings will be logged. Probably use the default constructor parameter to pass `cause` in instead of embedding `Throwable` in the String? 

``` scala 
extends Exception("Could not reach node.", cause)
```

Not nitpicking just trying to get this tiny stuff right so it makes debugging easier incase there is a blow up on the server.

Reply:
much better, it's fixed